docs/concept/rule_keys.soy

```diff
@@ -22,7 +22,7 @@
     </p>
     <p>
       RuleKeys are used to encapsulate all of the factors which may contribute to the output of a BuildRule into a single value - a hashcode.
-      If any RuleKey of a BuildRule has not changed, the output of that BuildRule cannot have changed.
+      If no RuleKey of a BuildRule has changed, the output of that BuildRule cannot have changed.
```

Comment:
instead of "no Rulekey" how about "none of the RuleKeys"

Reply:
That is indeed another way to say the same thing. I don't feel like it is obviously an improvement. If lots of people feel like @nemith though, I'm ok with that change.